### DISCUSSION TITLE
Unable to upgrade to new GitHub Actions for some repositories

### DISCUSSION THREAD

**Discussion by SamChou19815** [community/community#26112] [Actions]
<p>I already have the new GitHub actions enabled for some of my repos:</p>
<ul>
<li><a href="https://github.com/SamChou19815/website" rel="nofollow noopener">https://github.com/SamChou19815/website</a></li>
<li><a href="https://github.com/SamChou19815/samlang" rel="nofollow noopener">https://github.com/SamChou19815/samlang</a></li>
</ul>
<p>In the GitHub actions page I can see:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/177c40a894f5f981e67eec81e8213e38e672223e.png" data-download-href="/uploads/short-url/3lLbT0lWG50kq5sAFTojrqZUi7Q.png?dl=1" title="Screen Shot 2019-09-05 at 12.35.05.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/4290500/181095480-448e015c-d6e7-4eb8-a215-d409d48cecd1.png" alt="Screen Shot 2019-09-05 at 12.35.05.png" data-base62-sha1="3lLbT0lWG50kq5sAFTojrqZUi7Q" width="690" height="457" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/177c40a894f5f981e67eec81e8213e38e672223e_2_690x457.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/177c40a894f5f981e67eec81e8213e38e672223e_2_1035x685.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/1/177c40a894f5f981e67eec81e8213e38e672223e_2_1380x914.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/177c40a894f5f981e67eec81e8213e38e672223e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-09-05 at 12.35.05.png</span><span class="informations">1534×1017 85 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>However, in some of the repositories I can only see this:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/3b027c735288429961df0ee2f0cc8ea75e69598d.png" data-download-href="/uploads/short-url/8q1uPwFAdsKWAu43N7LUESz8ssB.png?dl=1" title="Screen Shot 2019-09-05 at 12.36.10.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/4290500/181095482-6d6311c6-2166-41d8-9a1f-2f347fb52c03.png" alt="Screen Shot 2019-09-05 at 12.36.10.png" data-base62-sha1="8q1uPwFAdsKWAu43N7LUESz8ssB" width="690" height="385" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/3b027c735288429961df0ee2f0cc8ea75e69598d_2_690x385.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/3b027c735288429961df0ee2f0cc8ea75e69598d_2_1035x577.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/3b027c735288429961df0ee2f0cc8ea75e69598d_2_1380x770.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/3b027c735288429961df0ee2f0cc8ea75e69598d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-09-05 at 12.36.10.png</span><span class="informations">3360×1875 591 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Example:</p>
<ul>
<li><a href="https://github.com/SamChou19815/cornell-api-libs" rel="nofollow noopener">https://github.com/SamChou19815/cornell-api-libs</a></li>
<li>
<a href="https://github.com/SamChou19815/db-prac" rel="nofollow noopener">https://github.com/SamChou19815/db-prac</a> (private)</li>
</ul>
<p>There is no way for me to upgrade the repo to new GitHub Actions. I even tried to create a old workflow file, and neither the new or old workflow can run.</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Hi Sam, sorry about that. I can click a button and upgrade all your repos to the new version. Sound good?</p>

**Comment by SamChou19815** [comment]
<p>Yes, that will be great!</p>

**Comment by mscoutermarsh** [comment]
<p>Done! try now</p>

**Comment by SamChou19815** [comment]
<p>Thanks. It works now!</p>

**Comment by rdmurphy** [comment]
<p>Sorry to add on here, but it seems my account is experiencing the same thing. :thinking: I can see the new workflows in new repositories, but all my older ones are stuck on the old version.</p>
